Penteract/Assets/Scripts/WinLose.cpp

```diff
@@ -16,6 +18,18 @@ GENERATE_BODY_IMPL(WinLose);
 void WinLose::Start() {
 	winCon = GameplaySystems::GetGameObject(winUID);
 	player = GameplaySystems::GetGameObject(playerUID);
+	enemies = GameplaySystems::GetGameObject(enemiesUID);
+	if (enemies->HasChildren()) {
```

Comment:
First, check if the enemies game object is not a nullptr, and once you know isn't it, then you can check if the GO have children or not.